proto/substrait/algebra.proto

```diff
@@ -131,14 +131,38 @@ message ReadRel {
       message ArrowReadOptions {}
       message OrcReadOptions {}
       message DwrfReadOptions {}
+      message TextReadOptions {
+        // The character(s) used to separate fields.  Common values are comma,
+        // tab, and pipe.
+        string field_delimiter = 1;
```

Comment:
Can this be multiple characters or is it restricted to one?

Reply:
Added clarifying text that multiple characters are allowed.